src/_data/docs_banner.yaml

```diff
@@ -1,7 +1,7 @@
 - type: "Webinar"
```

Comment:
Hi @Hasmin-AC! Could you please update or remove this, as it’s not a webinar? Here’s how it currently appears:

<img width="222" alt="Screenshot 2024-11-04 at 12 05 58" src="https://github.com/user-attachments/assets/4f3b5502-3750-43b3-8129-464c98f914c7">

Reply:
this should be removed